tests/admin_widgets/tests.py

```diff
@@ -1152,7 +1153,8 @@ def test_date_time_picker_shortcuts(self):
             By.CSS_SELECTOR, ".field-birthdate .datetimeshortcuts"
         )
 
-        now = datetime.now()
+        # Now is naive server time
+        now = localtime(datetime.now(tz=tz)).replace(tzinfo=None)
```

Comment:
Maybe we should simply skip this test with `@requires_tz_support` :thinking:

Reply:
Super -- I didn't know about this!